subsys/bluetooth/shell/bt.c

```diff
@@ -697,6 +697,35 @@ static int cmd_name(const struct shell *sh, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_appearance(const struct shell *sh, size_t argc, char *argv[])
+{
+	if (argc == 1) {
+		shell_print(sh, "Bluetooth Appearance: 0x%04x", bt_get_appearance());
+		return 0;
+	}
+
+	if (argc == 2) {
```

Comment:
You've made several choices that make the lines very long and the split lines hard to read. I'd start first here by inverting this condition and doing:
```
if (arc != 2) {
   shell_error(...);
  return 0;
}
```

Reply:
Well.. the split lines appear because we can't have helper functions. This function has multiple independent purposes (selected on by the argument count). I wanted to at least contain each functionality in it's own if-block. But it does not matter to me that much. I'll accept this change.